purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -117,6 +118,10 @@ fun Purchases.getOfferingsWith(
  * @param [onSuccess] Will be called after the purchase has completed
  * @param [onError] Will be called after the purchase has completed with error
  */
+@Deprecated(
+    "Replaced with purchaseProductWithOption",
+    ReplaceWith("purchaseProductWithOption")
```

Comment:
Did you try replacing using the Android Studio option to replace automatically? I want to make sure Android Studio replaces the function correctly automatically? I am saying because I think this needs to be but I am not sure

```
ReplaceWith("purchaseProductWithOption(activity, storeProduct, purchaseOption, onError, onSuccess)")
```

Reply:
Yeah, it's tricky, because then the dev would need to setup the `purchaseOption` variable beforehand. We could access the `storeProduct.purchaseOptions[0]` or something like that but that's assuming the offer which could not be the one the dev wants...
 
So I don't think this can be automatically done by Android Studio and it might make more sense to remove this entirely... Thoughts?

Reply:
Actually you were right, I just updated it. Thanks!